app/components/ShareButton/index.tsx

```diff
@@ -0,0 +1,43 @@
+import { Button, Flex, Icon } from '@webkom/lego-bricks';
+import { ShareIcon } from 'lucide-react';
+import { addToast } from 'app/reducers/toasts';
+import { useAppDispatch } from 'app/store/hooks';
+
+type Props = {
+  title: string;
+  url: string;
+};
+
+const ShareButton: React.FC<Props> = ({ title, url }) => {
+  const dispatch = useAppDispatch();
+  const handleShare = async () => {
+    if (navigator.share) {
+      try {
+        await navigator.share({ title, url });
+      } catch (error) {
+        console.error('Kunne ikke dele:', error);
+      }
+    } else {
+      {
+        navigator.clipboard.writeText(url);
+      }
+      dispatch(
+        addToast({
+          message: 'Link er kopiert til utklippstavlen',
+          type: 'success',
+          dismissAfter: 10000,
+        }),
+      );
+    }
+  };
+
+  return (
+    <Button onPress={handleShare}>
+      <Flex justifyContent="center">
+        <Icon name="share" iconNode={<ShareIcon />} size={20} />
```

Comment:
```suggestion
        <Icon iconNode={<Share2 />} size={20} />
```
`name` is not needed when using `iconNode`:)


Also imo. `Share2`<img width="39" alt="Screenshot 2025-03-05 at 22 12 55" src="https://github.com/user-attachments/assets/e5d62dd2-b3d7-4912-b9c5-0e45b05b1025" /> fits a bit better for this function

Reply:
Maybe would be nice with text on the button too, just to make it extra obvious what it does